cleanlab/count.py

```diff
@@ -224,6 +228,15 @@ def _num_label_issues_multilabel(
     return sum(issues_idx)
 
 
+def _reduce_issues(pred_probs, labels, num_classes):
+    pred = pred_probs.argmax(axis=1)
+    mask = pred == labels
+    if num_classes == 2:
+        # Set mask to False wherever pred_probs == 0.5 where num_classes == 2
+        mask = mask | ((pred_probs[:, 0] < 0.5 - EPSILON) | (pred_probs[:, 0] > 0.5 + EPSILON))
```

Comment:
I think what I suggested would be cleaner code:

mask = pred_probs[:,given_label] > 0.5 - EPS

that in future could generalize to more classes beyond K = 2

Reply:
even cleaner could be:

```
pred_probs[:,given_label] += EPS
pred = pred_probs.argmax(axis=1)
mask = pred == labels
```